src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -436,8 +436,8 @@ inline void JavaStringListToNativeStringVector(JNIEnv *env,
                                                jobject java_list,
                                                std::vector<std::string> *native_vector) {
   JavaListToNativeVector<std::string>(
-      env, java_list, native_vector, [](JNIEnv *env, jobject jstr) {
-        return JavaStringToNativeString(env, static_cast<jstring>(jstr));
+      env, java_list, native_vector, [](JNIEnv *env_arg, jobject jstr) {
```

Comment:
This is very weird based on my C++ knowledge, but compiler complains about it somehow.